templates/lambda.template.yaml

```diff
@@ -433,6 +444,132 @@ Resources:
 
           max_workers = 5
 
+          cf_client = boto3.client('cloudformation')
```

Comment:
In general, not a great fan of inline functions, but for now OK I guess.

Reply:
I note that several of the imports are not used in some of the other lambdas

Reply:
See https://gist.github.com/brinxmat/d3b15e198e3db612c611a8a95f682a5a where I attempt to fix the PEP8 warnings and restructure the code a bit